### ISSUE TITLE
travis is broken due to social_django

### ISSUE THREAD

**Issue by ThomasWaldmann** [nsupdate-info/nsupdate.info#295]
```
hostname: travis-worker-gce-org-prod4-8:d2b23fb7-6626-49d4-b419-82845427fd83
version: v2.5.0 https://github.com/travis-ci/worker/tree/da3a43228dffc0fcca5a46569ca786b22991979f
instance: testing-gce-73a884dc-3831-4491-851e-f957902c4a94:travis-ci-sugilite-trusty-1480960799

$ ./scripts/travis/install.sh
Reading package lists... Done
Building dependency tree       
Reading state information... Done
dnsutils is already the newest version.
e2fsprogs is already the newest version.
The following extra packages will be installed:
  bind9utils
Suggested packages:
  bind9-doc

The following NEW packages will be installed:
  bind9 bind9utils

0 upgraded, 2 newly installed, 0 to remove and 57 not upgraded.
Need to get 433 kB of archives.
After this operation, 1,634 kB of additional disk space will be used.

Get:1 http://us-central1.gce.archive.ubuntu.com/ubuntu/ trusty-updates/main bind9utils amd64 1:9.9.5.dfsg-3ubuntu0.10 [146 kB]
Get:2 http://us-central1.gce.archive.ubuntu.com/ubuntu/ trusty-updates/main bind9 amd64 1:9.9.5.dfsg-3ubuntu0.10 [287 kB]

Fetched 433 kB in 0s (27.4 MB/s)
Preconfiguring packages ...
Selecting previously unselected package bind9utils.
(Reading database ... 88286 files and directories currently installed.)
Preparing to unpack .../bind9utils_1%3a9.9.5.dfsg-3ubuntu0.10_amd64.deb ...
Unpacking bind9utils (1:9.9.5.dfsg-3ubuntu0.10) ...
Selecting previously unselected package bind9.
Preparing to unpack .../bind9_1%3a9.9.5.dfsg-3ubuntu0.10_amd64.deb ...
Unpacking bind9 (1:9.9.5.dfsg-3ubuntu0.10) ...
Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
Processing triggers for ureadahead (0.100.0-16) ...
Processing triggers for ufw (0.34~rc-0ubuntu2) ...
Setting up bind9utils (1:9.9.5.dfsg-3ubuntu0.10) ...
Setting up bind9 (1:9.9.5.dfsg-3ubuntu0.10) ...
Adding group `bind' (GID 122) ...
Done.

Adding system user `bind' (UID 114) ...
Adding new user `bind' (UID 114) with group `bind' ...
Not creating home directory `/var/cache/bind'.

wrote key file "/etc/bind/rndc.key"

 * Starting domain name service... bind9       Processing triggers for ureadahead (0.100.0-16) ...

Processing triggers for ufw (0.34~rc-0ubuntu2) ...
(Reading database ... 88371 files and directories currently installed.)
Removing ubuntu-minimal (1.325) ...
Removing resolvconf (1.69ubuntu1.1) ...
resolvconf stop/waiting
resolvconf.postrm: Reboot recommended
Purging configuration files for resolvconf (1.69ubuntu1.1) ...
Processing triggers for man-db (2.6.7.1-1ubuntu1) ...

Collecting Django==1.8.1

  Downloading Django-1.8.1-py2.py3-none-any.whl (6.2MB)

    100% |████████████████████████████████| 6.2MB 229kB/s 

Installing collected packages: Django

Successfully installed Django-1.8.1

Collecting django-debug-toolbar (from -r requirements.d/travis.txt (line 4))

  Downloading django_debug_toolbar-1.6-py2.py3-none-any.whl (205kB)

    100% |████████████████████████████████| 215kB 2.9MB/s 

Requirement already satisfied (use --upgrade to upgrade): pytest in /home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages (from -r requirements.d/travis.txt (line 5))

Collecting pytest-django (from -r requirements.d/travis.txt (line 6))

  Downloading pytest_django-3.1.2-py2.py3-none-any.whl

Collecting pytest-pep8 (from -r requirements.d/travis.txt (line 7))

  Downloading pytest-pep8-1.0.6.tar.gz

Collecting coveralls (from -r requirements.d/travis.txt (line 8))

  Downloading coveralls-1.1-py2.py3-none-any.whl

Collecting coverage (from -r requirements.d/travis.txt (line 9))

  Downloading coverage-4.3.1.tar.gz (372kB)

    100% |████████████████████████████████| 378kB 2.5MB/s 

Collecting sqlparse>=0.2.0 (from django-debug-toolbar->-r requirements.d/travis.txt (line 4))

  Downloading sqlparse-0.2.2-py2.py3-none-any.whl

Requirement already satisfied (use --upgrade to upgrade): Django>=1.8 in /home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages (from django-debug-toolbar->-r requirements.d/travis.txt (line 4))

Requirement already satisfied (use --upgrade to upgrade): py>=1.4.29 in /home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages (from pytest->-r requirements.d/travis.txt (line 5))

Collecting pytest-cache (from pytest-pep8->-r requirements.d/travis.txt (line 7))

  Downloading pytest-cache-1.0.tar.gz

Collecting pep8>=1.3 (from pytest-pep8->-r requirements.d/travis.txt (line 7))

  Downloading pep8-1.7.0-py2.py3-none-any.whl (41kB)

    100% |████████████████████████████████| 51kB 9.5MB/s 

Collecting docopt>=0.6.1 (from coveralls->-r requirements.d/travis.txt (line 8))

  Downloading docopt-0.6.2.tar.gz

Collecting requests>=1.0.0 (from coveralls->-r requirements.d/travis.txt (line 8))

  Downloading requests-2.12.4-py2.py3-none-any.whl (576kB)

    100% |████████████████████████████████| 583kB 1.9MB/s 

Collecting execnet>=1.1.dev1 (from pytest-cache->pytest-pep8->-r requirements.d/travis.txt (line 7))

  Downloading execnet-1.4.1-py2.py3-none-any.whl (40kB)

    100% |████████████████████████████████| 40kB 8.8MB/s 

Collecting apipkg>=1.4 (from execnet>=1.1.dev1->pytest-cache->pytest-pep8->-r requirements.d/travis.txt (line 7))

  Downloading apipkg-1.4-py2.py3-none-any.whl

Building wheels for collected packages: pytest-pep8, coverage, pytest-cache, docopt

  Running setup.py bdist_wheel for pytest-pep8 ... done

  Stored in directory: /home/travis/.cache/pip/wheels/18/26/e6/37ad77195c19a94c729fcd7fb3a6dadca805e34568510e4baa

  Running setup.py bdist_wheel for coverage ... done

  Stored in directory: /home/travis/.cache/pip/wheels/69/7e/69/d78768c4c56aabff723f474f11eacdf479c3454661f9eaa8a4

  Running setup.py bdist_wheel for pytest-cache ... done

  Stored in directory: /home/travis/.cache/pip/wheels/6d/22/a3/5b776623c29d057b5a71126cc3d7860ccfb08d5e4c377c3b72

  Running setup.py bdist_wheel for docopt ... done

  Stored in directory: /home/travis/.cache/pip/wheels/b2/16/5f/c33a2bb5f2dce71205f8e65cbfd05647d79d441282be31fd82

Successfully built pytest-pep8 coverage pytest-cache docopt

Installing collected packages: sqlparse, django-debug-toolbar, pytest-django, apipkg, execnet, pytest-cache, pep8, pytest-pep8, docopt, coverage, requests, coveralls

Successfully installed apipkg-1.4 coverage-4.3.1 coveralls-1.1 django-debug-toolbar-1.6 docopt-0.6.2 execnet-1.4.1 pep8-1.7.0 pytest-cache-1.0 pytest-django-3.1.2 pytest-pep8-1.0.6 requests-2.12.4 sqlparse-0.2.2

Obtaining file:///home/travis/build/nsupdate-info/nsupdate.info

Collecting dnspython (from nsupdate==0.12.0)

  Downloading dnspython-1.15.0-py2.py3-none-any.whl (177kB)

    100% |████████████████████████████████| 184kB 3.3MB/s 

Collecting netaddr (from nsupdate==0.12.0)

  Downloading netaddr-0.7.18-py2.py3-none-any.whl (1.5MB)

    100% |████████████████████████████████| 1.5MB 806kB/s 

Requirement already satisfied (use --upgrade to upgrade): django<1.9,>=1.8.1 in /home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages (from nsupdate==0.12.0)

Collecting django-bootstrap-form (from nsupdate==0.12.0)

  Downloading django-bootstrap-form-3.2.1.tar.gz

Collecting django-registration-redux<1.2 (from nsupdate==0.12.0)

  Downloading django-registration-redux-1.1.tar.gz (63kB)

    100% |████████████████████████████████| 71kB 8.4MB/s 

Collecting django-extensions (from nsupdate==0.12.0)

  Downloading django_extensions-1.7.5-py2.py3-none-any.whl (199kB)

    100% |████████████████████████████████| 204kB 3.9MB/s 

Collecting python-social-auth (from nsupdate==0.12.0)

  Downloading python_social_auth-0.3.1-py2-none-any.whl (150kB)

    100% |████████████████████████████████| 153kB 6.1MB/s 

Requirement already satisfied (use --upgrade to upgrade): requests in /home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages (from nsupdate==0.12.0)

Requirement already satisfied (use --upgrade to upgrade): six>=1.2 in /home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages (from django-extensions->nsupdate==0.12.0)

Collecting social-auth-core (from python-social-auth->nsupdate==0.12.0)

  Downloading social_auth_core-0.1.0-py2-none-any.whl (256kB)

    100% |████████████████████████████████| 266kB 4.3MB/s 

Collecting requests-oauthlib>=0.6.1 (from social-auth-core->python-social-auth->nsupdate==0.12.0)

  Downloading requests_oauthlib-0.7.0-py2.py3-none-any.whl

Collecting pyjwkest>=1.0.1 (from social-auth-core->python-social-auth->nsupdate==0.12.0)

  Downloading pyjwkest-1.3.2.tar.gz (41kB)

    100% |████████████████████████████████| 51kB 11.1MB/s 

Collecting PyJWT>=1.4.0 (from social-auth-core->python-social-auth->nsupdate==0.12.0)

  Downloading PyJWT-1.4.2-py2.py3-none-any.whl

Collecting python-openid>=2.2.5 (from social-auth-core->python-social-auth->nsupdate==0.12.0)

  Downloading python-openid-2.2.5.tar.gz (301kB)

    100% |████████████████████████████████| 307kB 3.4MB/s 

Collecting oauthlib>=1.0.3 (from social-auth-core->python-social-auth->nsupdate==0.12.0)

  Downloading oauthlib-2.0.1.tar.gz (122kB)

    100% |████████████████████████████████| 133kB 5.5MB/s 

Collecting pycryptodomex (from pyjwkest>=1.0.1->social-auth-core->python-social-auth->nsupdate==0.12.0)

  Downloading pycryptodomex-3.4.3.tar.gz (6.5MB)

    100% |████████████████████████████████| 6.5MB 212kB/s 

Collecting future (from pyjwkest>=1.0.1->social-auth-core->python-social-auth->nsupdate==0.12.0)

  Downloading future-0.16.0.tar.gz (824kB)

    100% |████████████████████████████████| 829kB 1.5MB/s 

Building wheels for collected packages: django-bootstrap-form, django-registration-redux, pyjwkest, python-openid, oauthlib, pycryptodomex, future

  Running setup.py bdist_wheel for django-bootstrap-form ... done

  Stored in directory: /home/travis/.cache/pip/wheels/36/cf/b0/82d31cf3695ac2dd24051cdaec372a91969de7aeec01906432

  Running setup.py bdist_wheel for django-registration-redux ... done

  Stored in directory: /home/travis/.cache/pip/wheels/da/f5/f2/44f55162556635e7d41f27863eee0c6cdb8deede40eed9b40f

  Running setup.py bdist_wheel for pyjwkest ... done

  Stored in directory: /home/travis/.cache/pip/wheels/51/6d/4c/2107156c3866d2c8633206c506e2d4fbadadc8a5e108e47c5d

  Running setup.py bdist_wheel for python-openid ... done

  Stored in directory: /home/travis/.cache/pip/wheels/0a/da/67/e9e68f4b5e03732dc17a545b4ce3ce84b4a9bef67253d4ff72

  Running setup.py bdist_wheel for oauthlib ... done

  Stored in directory: /home/travis/.cache/pip/wheels/e7/4e/b5/c36a97caf4408037eb35aeb1e0a80d95ea0c521a562c3488a3

  Running setup.py bdist_wheel for pycryptodomex ... done

  Stored in directory: /home/travis/.cache/pip/wheels/fc/c3/38/b1a190503ba9364b8613ec259785b954db04a414e02665aaed

  Running setup.py bdist_wheel for future ... done

  Stored in directory: /home/travis/.cache/pip/wheels/c2/50/7c/0d83b4baac4f63ff7a765bd16390d2ab43c93587fac9d6017a

Successfully built django-bootstrap-form django-registration-redux pyjwkest python-openid oauthlib pycryptodomex future

Installing collected packages: dnspython, netaddr, django-bootstrap-form, django-registration-redux, django-extensions, oauthlib, requests-oauthlib, pycryptodomex, future, pyjwkest, PyJWT, python-openid, social-auth-core, python-social-auth, nsupdate

  Running setup.py develop for nsupdate

Successfully installed PyJWT-1.4.2 django-bootstrap-form-3.2.1 django-extensions-1.7.5 django-registration-redux-1.1 dnspython-1.15.0 future-0.16.0 netaddr-0.7.18 nsupdate oauthlib-2.0.1 pycryptodomex-3.4.3 pyjwkest-1.3.2 python-openid-2.2.5 python-social-auth-0.3.1 requests-oauthlib-0.7.0 social-auth-core-0.1.0

1.34s$ $COVERAGE py.test $TEST_OPTS -k "$TEST_K" -m "not requires_sequential"

Traceback (most recent call last):

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/py/test.py", line 4, in <module>

    sys.exit(pytest.main())

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/config.py", line 47, in main

    config = _prepareconfig(args, plugins)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/config.py", line 156, in _prepareconfig

    pluginmanager=pluginmanager, args=args)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 745, in __call__

    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 339, in _hookexec

    return self._inner_hookexec(hook, methods, kwargs)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 334, in <lambda>

    _MultiCall(methods, kwargs, hook.spec_opts).execute()

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 613, in execute

    return _wrapped_call(hook_impl.function(*args), self.execute)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 250, in _wrapped_call

    wrap_controller.send(call_outcome)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/helpconfig.py", line 32, in pytest_cmdline_parse

    config = outcome.get_result()

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 280, in get_result

    _reraise(*ex)  # noqa

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 265, in __init__

    self.result = func()

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 614, in execute

    res = hook_impl.function(*args)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/config.py", line 910, in pytest_cmdline_parse

    self.parse(args)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/config.py", line 1067, in parse

    self._preparse(args, addopts=addopts)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/config.py", line 1038, in _preparse

    args=args, parser=self._parser)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 745, in __call__

    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 339, in _hookexec

    return self._inner_hookexec(hook, methods, kwargs)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 334, in <lambda>

    _MultiCall(methods, kwargs, hook.spec_opts).execute()

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 613, in execute

    return _wrapped_call(hook_impl.function(*args), self.execute)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 254, in _wrapped_call

    return call_outcome.get_result()

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 280, in get_result

    _reraise(*ex)  # noqa

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 265, in __init__

    self.result = func()

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 614, in execute

    res = hook_impl.function(*args)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/pytest_django/plugin.py", line 246, in pytest_load_initial_conftests

    _setup_django()

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/pytest_django/plugin.py", line 149, in _setup_django

    django.setup()

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/django/__init__.py", line 18, in setup

    apps.populate(settings.INSTALLED_APPS)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate

    app_config.import_models(all_models)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/django/apps/config.py", line 198, in import_models

    self.models_module = import_module(models_module_name)

  File "/opt/python/2.7.12/lib/python2.7/importlib/__init__.py", line 37, in import_module

    __import__(name)

  File "/home/travis/virtualenv/python2.7.12/lib/python2.7/site-packages/social/apps/django_app/default/models.py", line 1, in <module>

    from social_django.models import AbstractUserSocialAuth, UserSocialAuth, Nonce, Association, Code, DjangoStorage

ImportError: No module named social_django.models
```

**Comment by ThomasWaldmann** [comment]
likely in travis.txt something is not updated correctly:
```
pip install Django==$DJANGO
pip install -r requirements.d/travis.txt
pip install -e .
```

**Comment by ThomasWaldmann** [comment]
fixed by #296.
